fvm/accountV2Migration/AccountV2Migration.cdc

```diff
@@ -0,0 +1,89 @@
+access(all)
+contract AccountV2Migration {
+
+    access(all)
+    enum StorageFormat: UInt8 {
+        access(all)
+        case Unknown
+
+        access(all)
+        case V1
+
+        access(all)
+        case V2
+    }
+
+    access(all)
+    event Migrated(
+        addressStartIndex: UInt64,
+        count: UInt64
+    )
+
+    access(all)
+    resource Admin {
+        access(all)
+        fun setNextAddressStartIndex(_ nextAddressStartIndex: UInt64) {
+            AccountV2Migration.nextAddressStartIndex = nextAddressStartIndex
+        }
+
+        access(all)
+        fun setBatchSize(_ batchSize: UInt64) {
+            AccountV2Migration.batchSize = batchSize
+        }
+
+        access(all)
+        fun migrateNextBatch() {
+            AccountV2Migration.migrateNextBatch()
+        }
+    }
+
+    access(all)
+    let adminStoragePath: StoragePath
+
+    access(all)
+    var nextAddressStartIndex: UInt64
+
+    access(all)
+    var batchSize: UInt64
```

Comment:
If it is feasible, maybe we can have a `maxAddressIndex` corresponding to address index when we start migrating accounts. 

With something like a `maxAddressIndex`, batch migration can be disabled automatically.

Reply:
Good idea! Added in [336e2dc](https://github.com/onflow/flow-go/pull/6761/commits/336e2dc0d351ad015d87169acbd96cbf828abde6)